srcts/src/bindings/input/number.ts

```diff
@@ -49,10 +51,10 @@ class NumberInputBinding extends TextInputBindingBase {
     el;
   }
   receiveMessage(el: NumberHTMLElement, data: NumberReceiveMessageData): void {
-    if (hasOwnProperty(data, "value")) el.value = data.value;
-    if (hasOwnProperty(data, "min")) el.min = data.min;
-    if (hasOwnProperty(data, "max")) el.max = data.max;
-    if (hasOwnProperty(data, "step")) el.step = data.step;
+    if (hasOwnProperty(data, "value")) el.value = data.value ?? "";
+    if (hasOwnProperty(data, "min")) el.min = data.min ?? "";
+    if (hasOwnProperty(data, "max")) el.max = data.max ?? "";
+    if (hasOwnProperty(data, "step")) el.step = data.step ?? "";
```

Comment:
`el.value`, `el.min`, `el.max`, and `el.step` will now be set with `""` instead of `undefined`

Reply:
Don't know if it'd be better to just say it is a `string` and move on.

Ex: 
```suggestion
    if (hasOwnProperty(data, "value")) el.value = data.value as string;
    if (hasOwnProperty(data, "min")) el.min = data.min as string;
    if (hasOwnProperty(data, "max")) el.max = data.max as string;
    if (hasOwnProperty(data, "step")) el.step = data.step as string;
```

Reply:
Let's do this